uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/AbsencePeriodTest.php

```diff
@@ -293,6 +297,7 @@ public function startAndEndDatesDataProvider()
   public function startAndEndInvalidDatesDataProvider()
   {
     return [
+      ['2010-01-01', '2015-10-10'],
       ['2015-01-01', 'fdafdasfdsafdsafdsa'],
       ['232131232111', '2015-01-01'],
       ['2015-01-01', 12321321321],
```

Comment:
A lot of these tests are failing now probably because of the changes on AbsencePeriod BAO:
![captura de tela 2016-06-02 as 15 26 39](https://cloud.githubusercontent.com/assets/388373/15756189/86b44d86-28d6-11e6-9ad2-de898a394cbd.png)